integration/image/save_test.go

```diff
@@ -13,9 +13,9 @@ import (
 	"testing"
 	"time"
 
-	cerrdefs "github.com/containerd/errdefs"
 	"github.com/cpuguy83/tar2go"
 	containertypes "github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/image"
```

Comment:
This needs a rebase, and needs to use the new import paths for the api


After rebasing and updating the import paths,  it's also needed to run hack/vendor.sh because we now vendor the API module in the main module (don't ask 😆)